datasync/downloader/downloader_test.go

```diff
@@ -348,7 +348,7 @@ func (dl *downloadTester) CurrentBlock() *types.Block {
 
 	for i := len(dl.ownHashes) - 1; i >= 0; i-- {
 		if block := dl.ownBlocks[dl.ownHashes[i]]; block != nil {
-			if _, err := dl.stateDb.GetMemDB().Get(block.Root().Bytes()); err == nil {
```

Comment:
Is it the same behavior `dbm.getDatabase(StateTrieDB) == GetMemDB()`?

Reply:
Yes, but only if we're using Memory DB. See `NewMemoryDBManager()` and `getDatabase()`.

Reply:
Thanks.